packages/remix/rollup.config.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const babel = require("@rollup/plugin-babel").default;
+const path = require("path");
+
+const {
+  copyPublishFiles,
+  copyToPlaygrounds,
+  createBanner,
+  getOutputDir,
+} = require("../../rollup.utils");
+let { name: packageName, version } = require("./package.json");
+
+/** @returns {import("rollup").RollupOptions[]} */
+module.exports = function rollup() {
+  // Don't blow away remix magic exports on local builds, since they've
+  // already been configured by postinstall
+  if (process.env.REMIX_LOCAL_BUILD_DIRECTORY) {
+    return [];
+  }
+
+  let sourceDir = "packages/remix";
+  let outputDir = getOutputDir(packageName);
+  let outputDist = path.join(outputDir, "dist");
+  let outputEsmDist = path.join(outputDist, "esm");
+
+  return [
+    {
+      external() {
+        return true;
+      },
+      input: `${sourceDir}/index.ts`,
+      output: {
+        format: "cjs",
+        dir: outputDist,
+        banner: createBanner(packageName, version),
+      },
+      plugins: [
+        babel({
+          babelHelpers: "bundled",
+          exclude: /node_modules/,
+          extensions: [".ts", ".tsx"],
+        }),
+        copyPublishFiles(packageName),
+        copyToPlaygrounds(),
+      ],
+    },
+    {
+      external() {
+        return true;
+      },
+      input: `${sourceDir}/index.ts`,
+      output: {
+        format: "esm",
+        dir: path.join(outputDist, "esm"),
```

Comment:
This should be the unused `outputEsmDist` I guess?

```suggestion
        dir: outputEsmDist,
```

Reply:
I actually meant to delete the variable, no need for it here IMO